packages/e2e-tests/specs/demo.test.js

```diff
@@ -28,12 +28,12 @@ describe( 'new editor state', () => {
 		await visitAdminPage( 'post-new.php', 'gutenberg-demo' );
 	} );
 
-	it( 'content should load without making the post dirty', async () => {
+	it( 'content should load, making the post dirty', async () => {
 		const isDirty = await page.evaluate( () => {
 			const { select } = window.wp.data;
 			return select( 'core/editor' ).isEditedPostDirty();
 		} );
-		expect( isDirty ).toBeFalsy();
```

Comment:
@epiqueras @youknowriad @mcsf Why has this been changed???

Reply:
Initial edits should be considered a dirtying operation.

See https://github.com/WordPress/gutenberg/pull/16932#discussion_r315556523